frontend/src/store/TermsAndConditionsState.ts

```diff
@@ -0,0 +1,27 @@
+import { computed, ref } from 'vue';
+
+const termsVisibleState = ref(false);
+const termsClosableState = ref(false);
+
+export const isTermsVisible = computed({
+    get: () => termsVisibleState.value,
+    set: (value) => { termsVisibleState.value = value; }
+});
+
+export const isTermsClosable = computed({
```

Comment:
Hi Jota, shall we check if we need the isTermsClosable and toggleTermsCloseable? I feel for the terms and conditions from the profile bar, it's closable. And the one you had from the ManagePermission dashboard, that one is not closable.

Reply:
Hello Catherine! The `TermsAndCondition` component is used only in the `ManagePermissions` component. The `ProfileSidebar` will display the `TermsAndCondition` component again only if the terms button is clicked. Initially, the `isTermsClosable` property is set to false, requiring the user to either accept the terms or cancel and log out. When `isTermsClosable` is false, the "x" button will not be displayed. If the user clicks the terms button in the `ProfileSidebar`, the `TermsAndCondition` component will appear without the options to accept or cancel and can only be closed using the "x" button.

Please let me know if I misunderstood your comment

Reply:
yeah thanks Jota, I feel instead of a global state, I would prefer using a property. Please check the other comment, thanks!